server/controller/items.js

```diff
@@ -36,32 +36,32 @@ module.exports = {
       _.extend(item, newProps).save();
```

Comment:
how about using update method of sequelize model class? (in this case Item.update())